### ISSUE TITLE
Duplicate header and footer after Connect Twitter Account

### ISSUE THREAD

**Issue by nombrekeff** [forem/forem#9439]
**Description:**
I just connected my twitter account to my dev account, after twitter redirected back to dev the header and footer are duplicated, as seen in the screenshots. The dev console showed no errors.

**To Reproduce**

1. Log in to your dev account
2. Go to `https://dev.to/settings`
3. Click on Connect Twitter Account
4. After twitter redirects back, the header and footer is duplicate

**Screenshots**

![Captura de pantalla 2020-07-22 a las 14 54 44](https://user-images.githubusercontent.com/17043260/88179696-92b23000-cc2c-11ea-853a-173fbbed1bfd.png)
![Captura de pantalla 2020-07-22 a las 14 54 19](https://user-images.githubusercontent.com/17043260/88179706-96de4d80-cc2c-11ea-8bc1-29644db9d1ae.png)

**Desktop (please complete the following information):**

- OS: MacOS 10.15.5 (19F101)
- Browser: Chrome 
- Version: Versión 84.0.4147.89 (Build oficial) (64 bits)

**Additional context**
I tried disabling extensions, but the issue persisted.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @nombrekeff, I tried to replicate but was unsuccesful, I see you're on the dashboard page in the screenshots, did you go there after connecting the account?

**Comment by nombrekeff** [comment]
It redirected me automatically, I've noticed it redirects me to `https://dev.to/users/sign_up`. 

Another thing I just notice is It still shows the connect twitter button after connecting.

PD: I can replicate every single time if you need more info, I can also share a video of the process if you want to

**Comment by rhymes** [comment]
> PD: I can replicate it every single time if you need more info, I can also share a video of the process if you want to

that would be very helpful, thank you @nombrekeff !

**Comment by nombrekeff** [comment]
Sure, here you go: https://ufile.io/9zo4b1ru

If you need more info, I will be glad to help

**Comment by rhymes** [comment]
Thanks @nombrekeff - I took the liberty of downloading it and converting to a GIF so the issue is immediately clear to the contributors

![Grabacion-de-pantalla-2020-07-22](https://user-images.githubusercontent.com/146201/88197500-de6fd400-cc42-11ea-9a1a-628608ffe383.gif)

**Comment by nombrekeff** [comment]
Oh thanks, I forgot about doing that, my bad!

**Comment by nombrekeff** [comment]
Hey, I just realized I'm already connected on DEV with my twitter on another account. Maybe that is causing the issue.

**Comment by mikeydorje** [comment]
Closing as stale. Unable to reproduce by other contributors. No activity since November 2021.

If this issue occurs with current social authentication, please open a new issue with reproduction steps.
